src/templates/smfcfg.yaml.j2

```diff
@@ -9,18 +9,18 @@ info:
   description: SMF initial local configuration
 
 configuration:
-  smfDBName: {{ smf_database_name }}
+  smfDBName: {{ database_name }}
   enableDBStore: false
-  enableUPFAdapter: true
+  enableUPFAdapter: false
   debugProfilePort: 5001
   mongodb:
-    name: {{ default_database_name }}
-    url: {{ default_database_url }}
+    name: free5gc
```

Comment:
is this needed?

Reply:
Good catch, tested without and it seems like it's working